shared/actions/settings.js

```diff
@@ -66,6 +90,130 @@ function * saveNotificationsSaga (): SagaGenerator<any, any> {
   }
 }
 
+function * reclaimInviteSaga (invitesReclaimAction: InvitesReclaim): SagaGenerator<any, any> {
+  const {inviteId} = invitesReclaimAction.payload
+  try {
+    yield call(apiserverPostRpcPromise, {
+      param: {
+        endpoint: 'cancel_invitation',
+        args: [{key: 'invitation_id', value: inviteId}],
+      },
+    })
+    yield put(({
+      type: Constants.invitesReclaimed,
+      payload: undefined,
+    }: InvitesReclaimed))
+  } catch (e) {
+    console.warn('Error reclaiming an invite:', e)
+    yield put(({
+      type: Constants.invitesReclaimed,
+      payload: {errorText: e.desc + e.name, errorObj: e},
+      error: true,
+    }: InvitesReclaimed))
+  }
+  yield put(invitesRefresh())
+}
+
+function * refreshInvitesSaga (): SagaGenerator<any, any> {
+  try {
+    const json: ?{body: string} = yield call(apiserverGetRpcPromise, {
+      param: {
+        endpoint: 'invitations_sent',
+        args: [],
+      },
+    })
+
+    const results: {
+      invitations: [{
+        assertion: ?string,
+        ctime: number,
+        email: string,
+        invitation_id: string,
+        short_code: string,
+        type: string,
+        uid: string,
+        username: string,
+      }],
+    } = JSON.parse(json && json.body || '')
+
+    const acceptedInvites = []
+    const pendingInvites = []
+
+    results.invitations.forEach(i => {
+      const invite: Constants.Invitation = {
```

Comment:
I don't think this type declaration doesn't work, you have to `import type {Invitation} from '...constants'`